code/datums/objective.dm

```diff
@@ -1389,6 +1389,27 @@ ABSTRACT_TYPE(/datum/objective/conspiracy)
 
 		return 1
 
+/////////////////////////////////////////////////////////
+// Arcfiend Objectives                                 //
+/////////////////////////////////////////////////////////
+
+/datum/objective/specialist/powerdrain // this is basically just a repurposed vamp objective, but it should work.
+	var/powercount
```

Comment:
should probably called like power goal or something, since its a target not your count